app/Http/Controllers/Admin/WorkspaceController.php

```diff
@@ -25,9 +26,21 @@ public function __construct(WorkspaceRepository $workspaceRepository)
      */
     public function index()
     {
-        $workspaces = $this->workspace->getWorkspaces();
+        $workspaces = $this->workspace->with(['locations' => function ($query) {
+            $seats = DB::table('seats')
+                ->select(DB::raw('location_id, count(*) as total_seat'))
+                ->where(DB::raw('seats.usable'), '!=', config('site.default.unusable'))
+                ->orWhere('seats.usable')
+                ->groupBy('location_id');
 
-        return view('admin.workspace.list', compact('workspaces'));
+            $query->joinSub($seats, 'Total', function ($join) {
+                $join->on('locations.id', 'Total.location_id');
+            });
+        }])->get();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.CloseBraceNewLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/WorkspaceController.php</br>**Line**: 39</br>Closing bracket of array declaration must be on a new line